KitechRW/Kitech_backend
# Service subscription on invoice payment

#### what does this PR do?
- This PR adds the payment functionality and service subscription upon payment

#### Description of task to be completed
- Add an endpoint which will update invoice status from pending to paid
- Upon successful payment, service status should be changed to active

#### How should this be manually tested
- Clone the repo `git clone https://github.com/KitechRW/Kitech_backend.git`
- Run `cd Kitech_backend.git`
- Run ` git checkout ft-service-subscription-on-payment`
- Run `npm install`
- Use POSTMAN with `PATCH` method on this URL `http://localhost:3000/api/v1/invoice/:id`
- On successful payment invoice data should be displayed (refer to the screenshot).
#### Any background context you want to provide
- You need to be authenticated
- You role should be a `Manager`

#### What are the relevant trello story link
[Trello story link](https://trello.com/c/fHRkC1dR)
#### Screenshots (if appropriate)
![Screenshot from 2021-01-29 12-09-25](https://user-images.githubusercontent.com/69037813/106274137-724ada00-623c-11eb-92d3-bdbb9f4696a4.png)
- Only managers should update the invoice


#### Questions:
